src/Tizen.NUI/src/public/BaseComponents/TextLabel.cs

```diff
@@ -235,8 +235,7 @@ public string Text
             set
             {
                 SetValue(TextProperty, value);
-                selectorData?.Text?.Reset(this);
-                NotifyPropertyChangedAndRequestLayout();
+                NotifyPropertyChanged();
```

Comment:
This method trigger `PropertySet` event, I wonder what is different from `PropertyChanged` event, 
What means of `PropertySet`? is it no need to trigger when value was changed by data binding?

Reply:
thank you for review. `PropertySet` was added before the NUI XAML added. (NUI XAM has been copied from Xamarin.Forms open source) and it's role is exactly same as `PropertyChanged'. I can move all `NotifyPropertyChanged();` into `SetValue()` but I just want to get your guide whether removal of this `PropertySet' is better for usability? if so, then I will do ACR.

Reply:
@dongsug-song Yes, `PropertySet` event should be deprecated, if it is same with `PropertyChanged`, It make a confused and inconsistent

Reply:
@myroot 
OK, I see. I will proceed the ACR (https://code.sec.samsung.net/jira/browse/TCSACR-413) and will make it deprecated.